public/css/style.css

```diff
@@ -25,7 +27,10 @@ body {
   border: none;
   cursor: pointer;
   font: inherit;
-  box-shadow: 0 3px 6px 0 rgba(0,0,0,.05), 0 3px 6px 0 rgba(0,0,0,.05);
+  box-shadow: 0 3px 6px 0 rgba(0, 0, 0, 0.05), 0 3px 6px 0 rgba(0, 0, 0, 0.05);
+}
+.button:hover {
```

Comment:
We already have `cursor:pointer` applied above and we don't really want to override this. I was thinking of a hover effect that would alter the button color, opacity, and/or apply a small animation.

Reply:
Ahh, now I get what you meant. I'll be sure to fix them in a few days as I currently have finals for 3 consecutive days. Don't worry and thank you for pointing it out!

Reply:
You're welcome! No rush at all, take your time with this. Good luck with your exams 😄